controls/cis_al2023.yml

```diff
@@ -0,0 +1,2176 @@
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - package_audit_installed
+
+  - id: 5.2.1.2
+    title: Ensure auditing for processes that start prior to auditd is enabled (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - grub2_audit_argument
+
+  - id: 5.2.1.3
+    title: Ensure audit_backlog_limit is sufficient (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - grub2_audit_backlog_limit_argument
+
+  - id: 5.2.1.4
+    title: Ensure auditd service is enabled (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - service_auditd_enabled
+
+  - id: 5.2.2.1
+    title: Ensure audit log storage size is configured (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - auditd_data_retention_max_log_file
+      - var_auditd_max_log_file=6
+
+  - id: 5.2.2.2
+    title: Ensure audit logs are not automatically deleted (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - auditd_data_retention_max_log_file_action
+      - var_auditd_max_log_file_action=keep_logs
+
+  - id: 5.2.2.3
+    title: Ensure system is disabled when audit logs are full (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - auditd_data_retention_action_mail_acct
+      - auditd_data_retention_admin_space_left_action
+      - auditd_data_retention_space_left_action
+      - var_auditd_action_mail_acct=root
+      - var_auditd_admin_space_left_action=halt
+      - var_auditd_space_left_action=email
+
+  - id: 5.2.3.1
+    title: Ensure changes to system administration scope (sudoers) is collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_sysadmin_actions
+
+  - id: 5.2.3.2
+    title: Ensure actions as another user are always logged (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_suid_auid_privilege_function
+
+  - id: 5.2.3.3
+    title: Ensure events that modify the sudo log file are collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_sudo_log_events
+
+  - id: 5.2.3.4
+    title: Ensure events that modify date and time information are collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_time_adjtimex
+      - audit_rules_time_settimeofday
+      - audit_rules_time_clock_settime
+      - audit_rules_time_stime
+      - audit_rules_time_watch_localtime
+
+  - id: 5.2.3.5
+    title: Ensure events that modify the system's network environment are collected (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_networkconfig_modification
+      - audit_rules_networkconfig_modification_network_scripts
+
+  - id: 5.2.3.6
+    title: Ensure use of privileged commands is collected (Automated)
```

Comment:
Replace with "title: Ensure use of privileged commands are collected (Automated)".